satellite/console/consoleweb/server.go

```diff
@@ -348,13 +349,13 @@ func (server *Server) accountActivationHandler(w http.ResponseWriter, r *http.Re
 			zap.Error(err))
 
 		// TODO: when new error pages will be created - change http.StatusNotFound on appropriate one
-		server.serveError(w, r, http.StatusNotFound)
+		server.serveError(w, http.StatusNotFound)
 		return
 	}
 
 	if err = server.templates.activated.Execute(w, nil); err != nil {
 		server.log.Error("satellite/console/server: account activated template could not be executed", zap.Error(err))
-		server.serveError(w, r, http.StatusNotFound)
+		server.serveError(w, http.StatusNotFound)
```

Comment:
server.log.Error("satellite/console/server: account activated template could not be executed", zap.Error(Error.Wrap(err)))